tracing/zipkin/http.go

```diff
@@ -156,6 +156,12 @@ func HTTPServerTrace(tracer *zipkin.Tracer, options ...TracerOption) kithttp.Ser
 
 			if config.propagate {
 				spanContext = tracer.Extract(b3.ExtractHTTP(req))
+
+				if config.requestSampler != nil && spanContext.Sampled == nil {
```

Comment:
minor item... please swap the tested conditions.

Reply:
Done!